src/blant-utils.h

```diff
@@ -12,7 +13,7 @@ typedef struct nwhn { // this struct is needed for sorting
 struct orbitpair_bits {
 // NOTE that we store k-3 for k (thus k3), thus k=3 maps to 0, 8 maps to 5, and 10 maps to 7 (the highest value in 3 bits).
 #if MAX_K >= 9
-    unsigned int k3:4, ordinal: 38; // Kimia: I changed this from 54 to 38 since even at k=12 38 is enough
+    unsigned long long k3:4, ordinal: 38; // Kimia: I changed this from 54 to 38 since even at k=12 38 is enough
```

Comment:
Please rebase, I've changed this code in the meantime. k3 should definitely NOT be "long long" since it's only 3-4 bits, though ordinal might need it.